polynote-frontend/polynote/cell.js

```diff
@@ -985,16 +984,13 @@ export class CodeCell extends Cell {
     deactivateVim() {
         if (this.vim) {
             this.vim.dispose();
-            this.statusLine.innerHTML = '';
             delete this.vim;
         }
         this.hideVim();
     }
 
     hideVim() {
-        if (this.statusLine && !this.statusLine.contains(document.activeElement)) {
-            this.statusLine.classList.toggle('hide', true);
-        }
+        this.statusLine.innerHTML = '';
```

Comment:
We no longer need the `if (this.statusLine)` guard?

Reply:
I'm not convinced it was ever needed, because `this.statusLine` is always created in the `Cell` constructor.

Reply:
Oh I see, looks like the guard was left over from the initial implementation which didn't create it in the constructor. Nice! :)